src/actions/admin.ts

```diff
@@ -349,6 +369,20 @@ export function removePrivateSamlCertificate(): ActionFunc {
     });
 }
 
+export function removePublicLdapCertificate(): ActionFunc {
+    return bindClientFunc({
+        clientFunc: Client4.deletePublicLdapCertificate,
+        onRequest: AdminTypes.DELETE_LDAP_PUBLIC_REQUEST,
```

Comment:
You can also get rid of all of the `onRequest` parameters for all the functions added in this PR and then remove all the lines added to `src/action_types/admin.ts` since they wont be used

Reply:
I think we've left the `_SUCCESS` actions in some other places to allow other redux state to listen for when a request completes, but I agree on removing the `_REQUEST` one.